uuid_cache_dfn.csv

```diff
@@ -0,0 +1,3323 @@
+pkg-dfn150x150x100-4-pkg,a69ba545-da2c-4db5-ab96-fe7fc2e4880e
```

Comment:
This file is not related to this PR (probably added by mistake?)

Reply:
Yes, good catch, thanks.